src/platform/site-wide/sass/style.scss

```diff
@@ -4,7 +4,7 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 @import "~@department-of-veterans-affairs/formation/sass/shared-variables";
 @import "~@department-of-veterans-affairs/formation/sass/core";
 
-@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities.css";
+@import "~@department-of-veterans-affairs/css-library/dist/stylesheets/utilities";
```

Comment:
I'm removing the `.css` extension because otherwise `style.css` was rendering with this stylesheet hoisted to the top which was causing specificity issues. With the extension removed, Sass will compile this stylesheet in order.